docs/reference/run.md

```diff
@@ -981,12 +980,11 @@ or two examples of how to pass more parameters to that ENTRYPOINT:
 ## EXPOSE (incoming ports)
 
 The Dockerfile doesn't give much control over networking, only providing
-the `EXPOSE` instruction to give a hint to the operator about what
-incoming ports might provide services. The following options work with
-or override the Dockerfile's exposed defaults:
+the `EXPOSE` instruction to specify incoming ports that provide services.
```

Comment:
Not sure what you mean by "does nothing at all to the container". `EXPOSE` sets exposed ports on the container, You can see this by doing a `docker run` on an image built with `EXPOSE`, you will see the named ports listed as exposed ports. It's certainly true that they are not exposed _to the host_ unless `-p` or `-P` is used on `docker run`, but that's not the point. The point is that the documentation is making factually incorrect statements, and the changes in this PR corrects them.